### ISSUE TITLE
2018 approved absence not deducted from allowance

### ISSUE THREAD

**Issue by AsimPervez93** [timeoff-management/application#206]
I was testing some stuff out and I came across this possible bug.

As the test user, I requested 1 day in 2018 which was approved.  However, this does not come off the Allowance for the year as shown in the below screenshot.

![image](https://user-images.githubusercontent.com/17923501/33721830-0c5d2114-db60-11e7-8db4-362e5b5a3445.png)

**Comment by vpp** [comment]
Hi @AsimPervez93 

The calendr's statistics section always shows the current year.

Although it needs to be more flexible.

**Comment by AsimPervez93** [comment]
@vpp Ah I see, so come 2018, it would take it off the allowance correctly?

**Comment by vpp** [comment]
yes, but I am going to merge in "carry over" feature, that would allow employees to carry over lefteovers to next year.

**Comment by AsimPervez93** [comment]
Would it be possible to make the carry over feature optional?

I know most companies in my country do not allow employees to carry over leftovers to next year.

**Comment by vpp** [comment]
@AsimPervez93 in that case you just set the cary over value to be 0

**Comment by AsimPervez93** [comment]
@vpp Yeah that would work.  As long as the feature wouldn't mean that any leftover days would be automatically carried over.  This would be a good feature, just not very useful in my country.  Thanks for clarifying.

**Comment by enriquetoledano** [comment]
The carry over would be very useful in my company. Thanks!
